server/graphql/v2/object/Expense.js

```diff
@@ -42,6 +45,18 @@ const Expense = new GraphQLObjectType({
           };
         },
       },
+      payee: {
+        type: Account,
+        description: 'The account being paid by this expense',
+        async resolve(expense, _, req) {
+          // Set the permissions for account's fields
+          const canSeePrivateInfo = (await canViewExpensePrivateInfo(expense, req)).userLocation;
```

Comment:
Shouldn't this be `canSeeUserLocation`? Or maybe drop `.userLocation`?

Reply:
Simpler functions would be better indeed, I initially did that to return the full object in https://github.com/opencollective/opencollective-api/blob/94800abaf8fc5fb6c8d80c8e31079a1b2884feda/server/graphql/v1/types.js#L813 but let's not make the rest of the code more complex for just one endpoint.

Reply:
Simpler functions would be better indeed, I initially did that to return the full object in https://github.com/opencollective/opencollective-api/blob/94800abaf8fc5fb6c8d80c8e31079a1b2884feda/server/graphql/v1/types.js#L813 but let's not make the rest of the code more complex for just one endpoint.